hypha/apply/projects/views/payment.py

```diff
@@ -25,19 +25,23 @@
 
 
 @method_decorator(login_required, name='dispatch')
-class PaymentRequestAccessMixin:
+class PaymentRequestAccessMixin(UserPassesTestMixin):
```

Comment:
Rebuilt to use UserPassesTestMixin, looks ok?

Reply:
Looks good. We can also combine the three if statements into a single if inside the test_func.